DataProcessor/DataUnpacker.cpp

```diff
@@ -40,7 +40,7 @@ DataUnpacker::DataUnpacker(QObject *parent) : QObject(parent)
 {
     FILE* fp = fopen("../sc1-driver-io/sc1-data-format/format.json", "r"); // NOTE: Windows: "rb"; non-Windows: "r"
     if(fp == 0) {
-        fp = fopen("../solar-car-dashboard/sc1-data-format/format.json", "r"); // NOTE: Windows: "rb"; non-Windows: "r"
```

Comment:
This is a good catch. Is there anywhere else in the code where I/we copy pasted `solar-car-dashboard` as a directory when we shouldn't have?

Reply:
Looks like yes, here: https://github.com/badgerloop-software/sc1-driver-io/blob/4dcd2f7066e4cfe13119d77d9adddfb1e4c7eb47/main.cpp#L48

Do you also want to make this change in main.cpp at the same time as this PR? (although I guess it's sorta unrelated but probably not worth making a separate branch)

Reply:
This is the result of moving to cmake, right? I think this was a hacky workaround due to qmake putting outputs in an external `build-.../` directory. If we no longer need to leave the root project directory, let's just remove the `if (fp == 0) {` and just make it all one line: `FILE* fp = fopen("../sc1-data-format/format.json", "r"); // NOTE: Windows: "rb"; non-Windows: "r"`

Reply:
We can keep it this way, it will work as a backup if someone decided to build without build directory